### PR TITLE
GH#50841: typo and stylistic fixes

### PR DESCRIPTION
**Pull request by skopacz1**
Versions: 4.6+

#50841

This change fixes a typo (initator > initiator) and makes a few other stylistic updates for better alignment with the style guide.

Preview: https://51070--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-iscsi.html

### CODE REVIEW

File: modules/storage-persistent-storage-iscsi-enforcing-disk-quota.adoc [hunk 1]
```diff
@@ -9,5 +9,5 @@ is one persistent volume. Kubernetes enforces unique names for persistent
 volumes.
 
 Enforcing quotas in this way allows the end user to request persistent
-storage by a specific amount (e.g, 10Gi) and be matched with a
+storage by a specific amount (for example, 10Gi) and be matched with a
```
Comment (kelbrown20) [inline modules/storage-persistent-storage-iscsi-enforcing-disk-quota.adoc:12]: "10Gi" should be in back tics. Ex:

(for example, 10Gi) -> (for example, `10Gi`)

**Review by kelbrown20 — COMMENTED** [review]
Hello, great work! Other than that one nit, this LGTM 🚀

**Comment by kelbrown20** [comment]
Whoop!
/label peer-review-done

File: storage/persistent_storage/persistent-storage-iscsi.adoc [hunk 2]
```diff
@@ -31,8 +31,8 @@ By default, they are ports `860` and `3260`.
 
 [IMPORTANT]
 ====
-OpenShift assumes that all nodes in the cluster have already configured iSCSI
-initator, i.e. have installed `iscsi-initiator-utils` package and configured
+OpenShift assumes that all nodes in the cluster have already configured the iSCSI
+initiator, that is, they have installed the `iscsi-initiator-utils` package and configured
```
Comment (stevsmit) [inline storage/persistent_storage/persistent-storage-iscsi.adoc:35]: Who is they? Can we swap they with "users" ? As it currently reads, it seems like the "nodes" have some autonomy.

**Comment by skopacz1** [comment]
Changed the wording per feedback from @stevsmit, but I also plan to open a new ticket to update the wording of the entire admonition with the help of an SME.

File: storage/persistent_storage/persistent-storage-iscsi.adoc (hunk 2 above)
Comment (mburke5678) [reply storage/persistent_storage/persistent-storage-iscsi.adoc]: @skopacz1 When you [update the wording of the entire admonition](https://github.com/openshift/openshift-docs/pull/51070#issuecomment-1268423307), I think you need to s/OpenShift/{product-title}. I think we can merge this PR in the meantime.

**Pull request merged**